From 1b253237c146e09d667a77a3f6cd50a4002fb500 Mon Sep 17 00:00:00 2001
From: Potin Lai <potin.lai.pt@gmail.com>
Date: Fri, 26 Jul 2024 19:03:48 +0800
Subject: [PATCH 503/506] ARM: dts: aspeed: catalina: add extra ncsi properties

The Catalina dts file was removing some properties and compatible strings which
are not support in db-binding document and cause `CHECK_DTBS=y` check failed.

Add extra properties which only avaliable in Meta common patches.

- ncsi-ctrl,start-redo-probe
  - patch: common/recipes-kernel/linux/6.6/0173-NCSI-Add-propety-no-channel-monitor-and-start-redo-p.patch
- ncsi-package
  - patch: common/recipes-kernel/linux/6.6/0174-net-ncsi-specify-maximum-package-to-prob.patch

Signed-off-by: Potin Lai <potin.lai.pt@gmail.com>
---
 arch/arm/boot/dts/aspeed/aspeed-bmc-facebook-catalina.dts | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/arch/arm/boot/dts/aspeed/aspeed-bmc-facebook-catalina.dts b/arch/arm/boot/dts/aspeed/aspeed-bmc-facebook-catalina.dts
index 45428fec0699..47f770636a91 100644
--- a/arch/arm/boot/dts/aspeed/aspeed-bmc-facebook-catalina.dts
+++ b/arch/arm/boot/dts/aspeed/aspeed-bmc-facebook-catalina.dts
@@ -158,6 +158,8 @@ &mac3 {
 	pinctrl-names = "default";
 	pinctrl-0 = <&pinctrl_ncsi4_default>;
 	use-ncsi;
+	ncsi-ctrl,start-redo-probe;
+	ncsi-package = <1>;
 };
 
 &fmc {
-- 
2.44.2

